<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AlertWindow</class>
 <widget class="QWidget" name="AlertWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>587</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="maximumSize">
      <size>
       <width>400</width>
       <height>320</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border-radius:15px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>60</y>
        <width>400</width>
        <height>251</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>401</width>
         <height>251</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="contentLabel">
          <property name="minimumSize">
           <size>
            <width>370</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">border: none;
border-radius:0px;</string>
          </property>
          <property name="text">
           <string>内容</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="buttonsWidget" native="true">
          <property name="styleSheet">
           <string notr="true">QWidget{
border: none;
border-radius:0px;
}</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="sizeConstraint">
            <enum>QLayout::SetNoConstraint</enum>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>4</number>
           </property>
           <item>
            <widget class="QPushButton" name="confirmPushButton">
             <property name="minimumSize">
              <size>
               <width>90</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>90</width>
               <height>50</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton{
background-color: rgba(1, 131, 219,180);
color: rgb(255, 255, 255);
border: none;
border-radius:10px;
}

QPushButton:pressed{
background-color: rgb(1, 131, 219);
color: rgb(255, 255, 255);
border: none;
border-radius:10px;
}</string>
             </property>
             <property name="text">
              <string>确 定</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="cancelPushButton">
             <property name="minimumSize">
              <size>
               <width>90</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>90</width>
               <height>50</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton{
border:none;
border: 1px solid rgb(1, 131, 219);
border-radius:10px;
}
QPushButton::pressed{
	color: rgb(255, 255, 255);
	background-color: rgb(1, 131, 219);
}</string>
             </property>
             <property name="text">
              <string>取 消</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QLabel" name="titleLabel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>400</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(1, 131, 219);
color: rgb(255, 255, 255);
border-radius:0px;
border-top-left-radius:10px;
border-top-right-radius:10px;
border: none;</string>
      </property>
      <property name="text">
       <string>标题</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
